actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -438,6 +438,10 @@ def ssl?
       super || scheme == "wss"
     end
 
+    def trustworthy?
```

Comment:
Since this method is only used in the cookies, let's move it there. There is no need to expose it in the request.